packages/core/src/external/athenahealth/index.ts

```diff
@@ -181,6 +185,57 @@ class AthenaHealthApi {
     }
   }
 
+  async getPatientViaSearch({
+    cxId,
+    patientId,
+  }: {
+    cxId: string;
+    patientId: string;
+  }): Promise<PatientResource | undefined> {
+    const { log, debug } = out(`AthenaHealth search - AH patientId ${patientId}`);
+    try {
+      const data = {
+        _id: this.createPatientId(patientId),
+        "ah-practice": this.createPracticetId(this.practiceId),
+      };
+      const patienSearchtUrl = "/Patient/_search";
+      const response = await this.handleAxiosRequest(() =>
+        this.axiosInstanceFhirApi.post(patienSearchtUrl, this.createDataParams(data))
+      );
+      if (!response.data) throw new Error(`No body returned from ${patienSearchtUrl}`);
+      debug(`${patienSearchtUrl} resp: ${JSON.stringify(response.data)}`);
```

Comment:
nit: I think we can pass a function, so `JSON.stringify(response.data)` only gets executed if we're really debugging it

Reply:
Not sure I follow.

Reply:
leaving it open for the follow-up pr

Reply:
Will address in follow up utilities PR.